### PR TITLE
2022 interoperability portuguese

### PR DESCRIPTION
**Pull request by HakaCode**
Makes progress on #505

### CODE REVIEW

File: src/content/pt/2022/interoperability.md [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+results: https://docs.google.com/spreadsheets/d/1w3GzzTNeKxafFODmjDs6OC2dseNEDDKwUV8KeSgRI1Y/
+featured_quote: A interoperabilidade é um objetivo fundamental dos padrões, mas às vezes não conseguimos alcançá-lo totalmente. Este capítulo começará a fornecer uma atualização anual aos desenvolvedores sobre os esforços para se unirem e melhorar as coisas. Ele abordará o que é novo ou foi aprimorado em termos de interoperabilidade este ano e fornecerá um meio para os implementadores medirem os impactos ao longo do tempo.
+featured_stat_1: 309%
+featured_stat_label_1: The increase in sites using CSS `aspect-ratio` between April 2021 and September 2022
+featured_stat_2: 0.3%
+featured_stat_label_2: The percentage of sites using the newly interoperable `dialog` element as of September 2022
+featured_stat_3: 4%
+featured_stat_label_3: Mobile pages using recently interoperable CSS `containment`. This support is critical for Container Queries.
```
Comment (tunetheweb) [inline src/content/pt/2022/interoperability.md:18]: Can we translate these?

File: src/content/pt/2022/interoperability.md [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+
+## Compat 2021
+
+O Compat 2021 teve 5 principais áreas de foco:
+
+- Grid
+- Flexbox
+- Sticky Position
+- Transforms
+- Aspect Ratio
```
Comment (tunetheweb) [inline src/content/pt/2022/interoperability.md:41]: Probably better to leave these in their original English as technical terms but wonder should we add `lang="en"` spans?

```suggestion
- <span lang="en">Grid</span>
- <span lang="en">Flexbox</span>
- <span lang="en">Sticky Position</span>
- <span lang="en">Transforms</span>
- <span lang="en">Aspect Ratio</span>
```

File: src/content/pt/2022/interoperability.md [hunk 3]
```diff
@@ -0,0 +1,395 @@
...
+  width=709,
+  height=675
+  )
+}}
+
+Primeiro foi lançado no Chrome estável em julho de 2016. O Firefox lançou a segunda implementação em setembro de 2019. Neste ano, o Interop 2022 tomou a iniciativa de alinhar e garantir que, à medida que ele se torne universalmente disponível, comecemos em boa forma. Identificamos <a hreflang="en" href="https://wpt.fyi/results/css/css-contain?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-contain">235 testes</a>. <a hreflang="en" href="https://developer.apple.com/documentation/safari-release-notes/safari-15_4-release-notes">O Safari lançou o suporte para containment na versão estável 15.4</a> em março de 2022. Ao longo do ano, cada navegador melhorou o suporte e agora está universalmente disponível.
+
+{{ figure_markup(
+  caption="Número de páginas móveis que utilizam "containment" em suas folhas de estilo.",
```
Comment (tunetheweb) [inline src/content/pt/2022/interoperability.md:232]:
```suggestion
  caption='Número de páginas móveis que utilizam "containment" em suas folhas de estilo.',
```

**Review by tunetheweb — CHANGES_REQUESTED** [review]

File: src/content/pt/2022/interoperability.md (hunk 1 above)
Comment (HakaCode) [reply src/content/pt/2022/interoperability.md:18]: Yes, I was in doubt whether or not I could translate

Comment (HakaCode) [reply src/content/pt/2022/interoperability.md:18]: Yes, I was in doubt whether or not I could translate

File: src/content/pt/2022/interoperability.md (hunk 2 above)
Comment (HakaCode) [reply src/content/pt/2022/interoperability.md:41]: if there was no problem we can adjust, would that work for titles too?

**Review by tunetheweb — APPROVED** [review]
LGTM! 🎉

**Pull request merged**